shells/bash/Portfile

```diff
@@ -60,7 +62,23 @@ if {${subport} eq ${name}} {
                         bash51-004 \
                         rmd160  6d5536298d29c16bf1004f82ce6b6d0dd1312241 \
                         sha256  9aaeb65664ef0d28c0067e47ba5652b518298b3b92d33327d84b98b28d873c86 \
-                        size    3745
+                        size    3745 \
+                        bash51-005 \
+                        rmd160  a87d22af54ada099317ad0ceb2490f6bb7fba170 \
+                        sha256  cccbb5e9e6763915d232d29c713007a62b06e65126e3dd2d1128a0dc5ef46da5 \
+                        size    2577 \
+                        bash51-006 \
+                        rmd160  c78b9b2e6e69b073b6b42ea7f77f6c0996737fd4 \
+                        sha256  75e17d937de862615c6375def40a7574462210dce88cf741f660e2cc29473d14 \
+                        size    1412 \
+                        bash51-007 \
+                        rmd160  63f8cbba11151cc80efd53075137f802e87b92b6 \
+                        sha256  acfcb8c7e9f73457c0fb12324afb613785e0c9cef3315c9bbab4be702f40393a \
+                        size    2621 \
+                        bash51-008 \
+                        rmd160  1188a76917ca3602e4f10ece8b74cc500f5dd1aa \
+                        sha256  f22cf3c51a28f084a25aef28950e8777489072628f972b12643b4534a17ed2d1 \
+                        size    1821 \
```

Comment:
Is the trailing `\` syntactically valid here?

Reply:
I think in this case it doesn't cause any issues because it's at the end of the `if`-block. But I didn't intend to have a trailing `\` here and there is no need for so removing it - thanks for pointing that out!